### PR TITLE
chore: include MIT license in repo

### PR DESCRIPTION
**Pull request by bergzand**
# Description

Only the Apache license was included in the repo, while the README.md mentions both Apache and MIT. This in turn caused a discrepancy between what GitHub shows in the sidebar and what the README.md claims.

This PR renames the apache license file and adds the MIT license to the repo.

## Issues/PRs references

Mentioned off-line by @silkeh

## Change checklist

- [x] I have cleaned up my commit history and squashed fixup commits.
- [x] I have followed the [Coding Conventions](https://ariel-os.github.io/ariel-os/dev/docs/book/coding-conventions.html).
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.

### CODE REVIEW

**Comment by chrysn** [comment]
If we start having license files, it might be good to place them in their [reuse.software](https://reuse.software/spec-3.3/) spec places right away.

It would go well with SPDX headers at the long run, but those are clearly out of scope for this commit. (Maybe even moving the existing one is -- if this gets through quickly and moving to `LICENSES/${licensename}` is controversial, don't let this comment stop this PR).

**Comment by bergzand** [comment]
While I agree to follow this spec in general, local testing shows that [Licensee](https://github.com/licensee/licensee) ([used by GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license) doesn't detect license files in the `LICENSES` subdirectory. For now I want to stick with something that is picked up by GitHub so all our licenses show up in the project metadata

**Comment by bergzand** [comment]
> While I agree to follow this spec in general, local testing shows that [Licensee](https://github.com/licensee/licensee) ([used by GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#detecting-a-license)) doesn't detect license files in the `LICENSES` subdirectory. For now I want to stick with something that is picked up by GitHub so all our licenses show up in the project metadata

I want to stick with a mechanism that is picked up by GH, so that the set of licenses under which contributors make their contributions is unambiguous.

There's an open issue for REUSE compliance with Licensee to fix this: https://github.com/licensee/licensee/issues/737, I suggest we move the license files to `LICENSES/` as soon as that is resolved (and GH deployed it)

**Comment by chrysn** [comment]
Fine with me. I prefer not having GitHub dictate conventions, but it's your call, and a practical outcome.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Welcome and thanks for your interest in contributing to Ariel OS! We appreciate
     - Please indicate a ready-for-review PR with the button in GitHub
     - We use [Conventional Commits](https://www.conventionalcommits.org/en)
 - Please check our [Coding Conventions](https://ariel-os.github.io/ariel-os/dev/docs/book/coding-conventions.html)
-- Ariel OS is dual [licensed](https://github.com/ariel-os/ariel-os/blob/main/LICENSE) under the Apache-2.0 and MIT licenses
+- Ariel OS is dual licensed under the [Apache-2.0](./LICENSE-APACHE) and [MIT](./LICENSE-MIT) licenses
```
Comment (chrysn) [inline CONTRIBUTING.md:13]: I did check that there is no place where CONTRIBUTING.md is inlined in the code base.

File: LICENSE-MIT
Comment (chrysn) [inline LICENSE-MIT:1]: This is indeed the MIT license as per <https://spdx.org/licenses/MIT.html>; text that is in the SPDX version in blue (the "(including the next paragraph)") is omitted, and that's what blue meens on SPDX.

**Review by chrysn — APPROVED** [review]
Both comments are merely to point out that checks happened; LGTM.

**Pull request merged**